tools/c7n_salactus/c7n_salactus/worker.py

```diff
@@ -108,12 +108,15 @@ def patch_ssl():
 
 DEFAULT_TTL = 60 * 60 * 48
 
+# Default size of the bucket before checking for inventory
+DEFAULT_INVENTORY_BUCKET_SIZE_THRESHOLD = 100000
```

Comment:
might as well making it an environ config rather a pure constant

Reply:
Done - `SALACTUS_INVENTORY_THRESHOLD`